edit-post/components/sidebar/settings-header/style.scss

```diff
@@ -17,6 +17,18 @@
 	color: $dark-gray-900;
 	@include square-style__neutral;
 
+	// This pseudo-element "duplicates" the tab label and sets the text to bold.
+	// This ensures that the tab doesn't change width when selected.
```

Comment:
Good description; I'll add a link to this PR (just in case `git blame` eventually doesn't lead someone back to here) for context.